src/cli.js

```diff
@@ -43,112 +43,138 @@ async function parseArgumentsIntoOptions(rawArgs) {
 export async function cli(args) {
 	const options = await parseArgumentsIntoOptions(args);
 
+	// drop out if not logged in for certain commands
+	const userCommands = ['init', 'recs', 'recommendation', 'recommendations', 'secret', 'secrets', 'logout', 'whoami', 'org-access'];
```

Comment:
`recs sync` command will need to function without a logged in user (ie. when it runs in the action)

A simple solution would be to check if `--secrets-ci` is passed in

Reply:
Ah... yup. I'll have to wait until that change has been merged in I suppose.